contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -379,15 +373,17 @@
           });
         });
       }),
-      copyToClipboard: withChangeTracker(function(id__in, changeTracker) {
+      copyToClipboard: withChangeTracker(function(ids, changeTracker) {
+        const nodes = this.getContentNodes(ids);
```

Comment:
This getter isn't defined here yet:

`vue.runtime.esm.js:619 [Vue warn]: Error in v-on handler (Promise/async): "TypeError: this.getContentNodes is not a function"`

Reply:
I found all of this testing the clipboard. After I added `getContentNodes` to the file locally to see if it was the only thing stopping clipboard from working, I get this error:

`vue.runtime.esm.js:619 [Vue warn]: Error in v-on handler (Promise/async): "DataError: Failed to execute 'only' on 'IDBKeyRange': The parameter is not a valid key."`

Reply:
That error will happen when `undefined` gets passed in as a key parameter to IndexedDB. Guessing the getContentNodes got lost in the rebase. Thanks for testing here.

Reply:
Managed to replicate and resolve!